.github/workflows/example-docker.yml

```diff
@@ -19,7 +19,9 @@ jobs:
       options: --user 1001
     steps:
       - uses: actions/checkout@v4
-      - uses: cypress-io/github-action@v6
+      # replace by the following for regular use:
```

Comment:
```suggestion
      # replace with the following for regular use:
```

Reply:
"by" and "with" are often interchangeable. "with" is generally used if something is broken.

I don't want to hold up this PR because of a discussion about grammar in a comment appearing in an example however, so I have taken your suggestion on board.

From https://www.merriam-webster.com/dictionary/replace:
![image](https://github.com/cypress-io/github-action/assets/66998419/30ca9d46-7413-40d9-9765-02f8f5184414)